### PR TITLE
[tune] wandb - WandbLogger now also accepts wandb.data_types.Video

### PR DESCRIPTION
**Pull request by daanklijn**
## Why are these changes needed?

Currently the WandbLogger does not accept `wandb.data_types.Video`'s as they are stripped away by the `_is_allowed_type` method. To fix this I also check for the `wandb.data_types.Video` and let the method return `True` if it finds this data type. I also make sure that Iterables containing this data type are accepted (as `wandb.log` also accepts this). 

More about logging videos using wandb can be read [here](https://docs.wandb.ai/library/log#log-a-video).

## Checks

- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by richardliaw — APPROVED** [review]
Looks great!

**Review by richardliaw — COMMENTED** [review]
Sorry for the slow reply! Would you be willing to add a quick test?

**Comment by daanklijn** [comment]
Thanks for the review @richardliaw. I am quite busy atm but I'll be happy to write a test for it somewhere in the next few weeks. If you think this shoud be merged sooner, please don't hesitate get someone else to do it.

**Comment by richardliaw** [comment]
@daanklijn no problem -- thanks a bunch for the patch!

File: python/ray/tune/integration/wandb.py [hunk 1]
```diff
@@ -22,13 +23,17 @@
 
 WANDB_ENV_VAR = "WANDB_API_KEY"
 _WANDB_QUEUE_END = (None, )
+_VALID_TYPES = (Number, wandb.data_types.Video)
+_VALID_ITERABLE_TYPES = (wandb.data_types.Video)
```
Comment (hartikainen) [inline python/ray/tune/integration/wandb.py:27]: This should probably be:
```python
_VALID_ITERABLE_TYPES = (wandb.data_types.Video, )
```

**Pull request merged**